internal/store/verticalpodautoscaler.go

```diff
@@ -19,6 +19,7 @@ package store
 import (
 	"context"
 
+	k8sautoscaling "k8s.io/api/autoscaling/v1"
```

Comment:
```suggestion
	autoscalingv1 "k8s.io/api/autoscaling/v1"
```

Can we refactor this to use a similar approach like metav1 below?

Reply:
I copied it from the ksm code somewhere, but yes, no problem. I will make the change.